nss-day-cohort-22/Class-Website
# Renamed classmate image files to reflect each person's name

## Link to Ticket

no direct ticket

## Description of Proposed Changes

- Renamed all the image files in the classmates folder to reflect each classmates name

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout gl-AddingNamesToImageFiles

```
Look in images/classmates/ folder and make sure the photos are named for each person, and not generic IMG_2233.jpg style names
Load page via HTTP-SERVER and check that my photo still works. :) 

## Impacted Areas in Application

List general components of the application that this PR will affect:

* images/classmates
* cohort.json

## Mentions @username

Tag users that need to review this code
